client/src/js/services/GeneralLedgerService.js

```diff
@@ -0,0 +1,26 @@
+angular.module('bhima.services')
+  .service('GeneralLedgerService', GeneralLedgerService);
+
+// Dependencies injection
+GeneralLedgerService.$inject = ['$http', 'util'];
+
+/**
+ * General Ledger Service
+ * This service is responsible of all process with the General ledger
+ */
+function GeneralLedgerService($http, util) {
+  'use strict';
```

Comment:
Just for your information, you also don't need to use `use strict` on the client, since our gulp build script will add it.  It's not a problem to use it though.